redcap_bridge/test_redcap/testfiles/compression_test/original_record.csv

```diff
@@ -0,0 +1,8 @@
+modality___a,modality___b,modality___c,modality___d,a,b,c,d,e,f,cat___z,cat___x,cat___s
+ 1,1,1,0,0,0,0,0,0,0,1,1,1
```

Comment:
currently all lines are identical here. The test would be more powerful if it would cover different cases (i.e. also where no choice was selected for a modality)